content/posts/newsletter-july-2024.md

```diff
@@ -0,0 +1,36 @@
+title: Read the Docs Newsletter - July 2024
+date: July 9, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community
+authors: Eric Holscher
+status: published
+image: /images/july-2024.jpg
+image_credit: Photo by <a href="https://unsplash.com/@dyc251?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Diana Yohannes</a> on <a href="https://unsplash.com/photos/a-tree-in-front-of-a-sunset-vmNdWNX3xpU?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+We missed a couple of newsletter updates because of travel and other commitments, so we're back with a combined update for the last couple months.
+
+## News and updates
```

Comment:
All the `build.tools` were updated to their latest versions: https://github.com/readthedocs/readthedocs.org/pull/11386